src/util/manifest.js

```diff
@@ -83,3 +84,11 @@ export function getManifestId(manifestData: ExtensionManifest): string | void {
   return manifestData.applications ?
     manifestData.applications.gecko.id : undefined;
 }
+
+function stripBOM(content) {
```

Comment:
This needs tests. There's a manifest with a BOM [attached here](https://github.com/mozilla/web-ext/issues/1013#issuecomment-319105758) that you can use for the tests.